osgpkitools/incommon_request.py

```diff
@@ -296,7 +220,7 @@ def submit_request(config, restclient, hostname, cert_csr, sans=None):
             response_data = json.loads(response_text)
             response_data = response_data['sslId']
         else:
-            raise AuthenticationFailureException(response.status, "Failed connection to InCommon API.")
+            raise AuthenticationFailureException(response.status, "Connection failure to InCommon API. Check your authentication credentials.")
```

Comment:
Oh I see that the enrollment request also assumed the error was an authentication failure, but there could be other kinds of failures.   I think including the http response reason could be useful here too.

Reply:
Or you could check first for a 401 status before assuming it is an authentication failure.

If you give this version a bad username, you get the following message:
```
Got an exception AuthenticationFailureException
Connection failure to InCommon API. Check your authentication credentials.
Check your authentication credentials.
```
It would be better to follow the format of other error messages: "osg-incommon-cert-request: Connection failure to InCommon API. Check your authentication credentials."

Reply:
Fixed.

Reply:
I also removed the print_exception_message from utils.py